lib/block-patterns/heading.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Heading.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => __( 'Heading' ),
+	'categories'  => array( 'text' ),
+	'content'     => '<!-- wp:heading {"align":"wide","style":{"typography":{"fontSize":"48px","lineHeight":"1.1"}}} -->
+	<h2 class="alignwide" style="font-size:48px;line-height:1.1">' . __( "We're a studio in Berlin with an international practice in architecture, urban planning and interior design. We believe in sharing knowledge and promoting dialogue to increase the creative potential of collaboration." ) . '</h2>
```

Comment:
Do we need to escape the text strings in all of these patterns? The current bundled patterns are unescaped, but many themes do escape them (Twenty Twenty-One, [for example](https://github.com/WordPress/twentytwentyone/blob/trunk/inc/block-patterns.php)).

Reply:
I'm not sure if it's needed, as you said the core ones are unescaped, but I escaped these just in case.